pkg/bitwarden/cli.go

```diff
@@ -0,0 +1,118 @@
+package bitwarden
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+)
+
+type cliClient struct {
+	username string
+	password string
+	sync.Mutex
+	session    string
+	savedItems []Item
+	run        func(cmd string, args ...string) ([]byte, error)
+}
+
+func newCliClient(username, password string) (Client, error) {
```

Comment:
Why do you need `newCliClient` and `NewClient`?

Reply:
`NewClient`: the constructor for the client interface. out-of-pkg caller uses it.
`newCliClient`: the constructor for the cli-implementation. Used by `NewClient`. I guess you want me to use the `cliClient{}` directly. I think hiding loginAndList() call in `newCliClient` looks nicer.

One day, if we switch to e.g., rest-implementation, whose constructor would be `newRestClient` and `NewClient` will call it.